src/libraries/Common/src/Interop/Interop.Brotli.cs

```diff
@@ -19,7 +19,7 @@ internal static class Brotli
             ref nuint availableOut, byte** nextOut, out nuint totalOut);
 
         [DllImport(Libraries.CompressionNative)]
-        internal static extern unsafe bool BrotliDecoderDecompress(nuint availableInput, byte* inBytes, ref nuint availableOutput, byte* outBytes);
+        internal static extern unsafe int BrotliDecoderDecompress(nuint availableInput, byte* inBytes, nuint* availableOutput, byte* outBytes);
```

Comment:
Should we make all of PInvokes in this file to return `int` to return consistency? 

Also, would it make sense to use `BOOL` to make it clear that this is returning boolean value?

Reply:
Are you talking about that `Interop.BOOL` enum?

Reply:
I believe so.